malos/heartbeat.proto

```diff
@@ -13,6 +13,7 @@ import "../malos/driver.proto";
 message Beat {
   repeated EnumMalosEyeDetectionType services = 1;
```

Comment:
For the future: It's better to use singulars for repeated fields.

Sounds a bit counter-intuitive but then in code it's better to write:
   beat.add_integration() than beat->add_integrations.

But this is debatable, as then you need to do "integration_size" as opposed to "integrations_size()".